WooCommerce/Classes/ViewRelated/Products/Edit Product/Product+ProductForm.swift

```diff
@@ -34,4 +34,22 @@ extension Product {
             return categoriesNames.joined(separator: ", ")
         }
     }
+
+    /// Returns a comma separated string with each tags names.
+    /// Returns `nil` if the product doesn't have any associated tag
+    /// iOS 13+ set a specific locale on to properly format the list with the `ListFormatter` class
+    func tagsDescription(using locale: Locale = .autoupdatingCurrent) -> String? {
+        guard tags.isNotEmpty else {
+            return nil
+        }
+
+        let tagsNames = tags.map { $0.name.capitalized }
```

Comment:
just wondering, why don't we show the tags with its own capitalization?

Reply:
👍 Updated 011a3cdc6a4c8a2495d5bb9463a21222fae790ed, now we always show tags with its own capitalization, I think it is better to show the tags as they were inserted.


<img src="https://user-images.githubusercontent.com/495617/87531053-71cc6680-c691-11ea-86c6-51d6c89d9a87.png" width=300 />